endpoints/openrtb2/video_auction.go

```diff
@@ -79,7 +81,38 @@ func (deps *endpointDeps) VideoAuctionEndpoint(w http.ResponseWriter, r *http.Re
 		CookieFlag:    pbsmetrics.CookieFlagUnknown,
 		RequestStatus: pbsmetrics.RequestStatusOK,
 	}
+
+	debugQuery := r.URL.Query().Get("debug")
+	cacheTTL := int64(3600)
+	if deps.cfg.CacheURL.DefaultTTLs.Video > 0 {
+		cacheTTL = int64(deps.cfg.CacheURL.DefaultTTLs.Video)
+	}
+	debugLog := exchange.DebugLog{
+		EnableDebug: strings.EqualFold(debugQuery, "true"),
+		CacheType:   prebid_cache_client.TypeXML,
+		TTL:         cacheTTL,
+	}
+
 	defer func() {
+		if len(debugLog.CacheKey) > 0 && vo.VideoResponse == nil {
+			debugLog.Data = fmt.Sprintf("<!--\n%s\n\nNo response created\n-->", debugLog.Data)
```

Comment:
Go tooling reports there is no unit test coverage for inside of the if statement. Please add a test case.

Reply:
Added a unit test for the failure case. Should be covered now.

Reply:
Looks good now. Thank you.